mods/pageviews.js

```diff
@@ -83,7 +60,10 @@ var tableSchemas = {
             granularity: 'string',
             // the hourly timestamp will be stored as YYYYMMDDHH
             timestamp: 'string',
-            views: 'int'
+            views: 'int',
+            // store this as a string because it's too big for an int
+            // and long/bigint are not supported in RESTBase at this time
+            v: 'string'
```

Comment:
I'm using "v" for two reasons:

1. Cassandra stores column names with each row, so I've become very cautious about descriptive names
2. This is a helper column, and hopefully temporary.  It seemed from the discussions yesterday that even though RESTBase didn't support removing a column name, we could hack the table such that eventually we end up with views: 'long' instead of views: 'int'.  I get that having both views and v, with nulls in one or the other doesn't store any more data than just having a single column, but I also hate looking at code that has two columns which do the same thing.